script/openhexa.sh

```diff
@@ -82,6 +83,7 @@ function is_backend_reachable() {
 }
 
 function is_db_accepting_connexion() {
+  # TODO replace with get_config_or_default
```

Comment:
Do you want to address this TODO here or in the future ?

Reply:
later